azure-pipelines.yml

```diff
@@ -0,0 +1,91 @@
+parameters:
+- name: module_names
+  type: object
+  default:
+  - nidigital
```

Comment:
Add a comment about why you chose this particular order.

Reply:
Based on time taken by the modules.
NIDCPower/nitclk are taking around 15 mins.
nifgen is taking 5 mins
dcpower, dmm, modinst, scope are taking around 2 mins.
se(Swith Executive), Switch is taking around 1 min.

Reply:
That's a brittle comments. Are the times at least consistent? Why in the world does `nidcpower` take 15 minutes?!

Reply:
I think he meant nidigital and not nidcpower takes 15 mins, which is due to `test_self_calibrate` taking a long time.